src/scenes/CollectionGalleryPage/CollectionGalleryPage.tsx

```diff
@@ -27,6 +32,15 @@ function CollectionGalleryPage({ collectionId, username }: CollectionGalleryPage
     });
   }, [username, collectionId, track]);
 
+  const query = useFragment(
```

Comment:
can you explain again the reasoning behind why queries should be made at the top-level component then prop-drilled as needed?

is it because other components on the page may grab data from the query in the future?